rdr_service/resource/tasks.py

```diff
@@ -32,16 +32,18 @@ def batch_rebuild_participants_task(payload):
     batch = payload['batch']
 
     logging.info(f'Start time: {datetime.utcnow()}, batch size: {len(batch)}')
+    # logging.info(json.dumps(batch, indent=2))
```

Comment:
Suggest maybe keeping this enabled as logging.debug data, instead of info?

Reply:
We don't want to ever show the payloads in Production, so I would be careful and not enable this in Prod.